### PR TITLE
[Chromium] Implement ColorPicker

### PR DESCRIPTION
**Pull request by mshin-wolvic**

### CODE REVIEW

**Review by svillar — CHANGES_REQUESTED** [review]
Looks great, I just have a minor comment suggesting splitting the commit in two. See inline

File: app/src/common/chromium/com/igalia/wolvic/browser/api/impl/PromptDelegateImpl.java [hunk 1]
```diff
@@ -490,4 +493,66 @@ public void hide(boolean sendsCancelMessage) {
...
+        public WSession.PromptDelegate.PromptResponse confirm(@NonNull final String color) {
+            markComplete();
+            mListener.onColorChanged(Color.parseColor(color));
+            return new PromptResponseImpl();
+        }
+
+        @NonNull
+        @Override
+        public WSession.PromptDelegate.PromptResponse dismiss() {
+            if (!isComplete())
```
Comment (zakharvoit) [inline app/src/common/chromium/com/igalia/wolvic/browser/api/impl/PromptDelegateImpl.java:553]: Not sure, but probably we need markComplete() here?

File: app/src/common/chromium/com/igalia/wolvic/browser/api/impl/PromptDelegateImpl.java [hunk 2]
```diff
@@ -237,7 +240,7 @@ public SelectPopup.Ui create(Context windowContext, Callback<int[]> selectionCha
         }
     }
 
-    public class ChoicePrompt extends BasePromptImpl implements  WSession.PromptDelegate.ChoicePrompt {
+    public class ChoicePrompt extends BasePromptImpl implements WSession.PromptDelegate.ChoicePrompt {
```
[the remark this answers is not in the document]
Comment (mshin-wolvic) [reply app/src/common/chromium/com/igalia/wolvic/browser/api/impl/PromptDelegateImpl.java]: oop. Android Studio automatically adjusted the coding format. I removed the unrated clean code.

File: app/src/common/chromium/com/igalia/wolvic/browser/api/impl/PromptDelegateImpl.java (hunk 1 above)
Comment (mshin-wolvic) [reply app/src/common/chromium/com/igalia/wolvic/browser/api/impl/PromptDelegateImpl.java]: No problems were found when testing, but added this code by referencing other code. thanks!

**Review by svillar — APPROVED** [review]

**Pull request merged**